server/util/json-ld.js

```diff
@@ -7,3 +9,52 @@ export function objToJsonLd<T>(obj: T, type: string, root: boolean = true) {
   } : { '@type': type };
   return Object.assign({}, jsonObj, jsonLdAddition);
 }
+
+export function contentLd(content) {
+  return objToJsonLd({
+    headline: content.headline,
+    author: personLd(content.author),
+    image: imageLd(content.thumbnail),
+    datePublished: content.datePublished,
+    dateModified: content.datePublished,
+    publisher: orgLd(wellcomeCollection)
+  }, 'Article');
+}
+
+export function museumLd(museum) {
+  const openingHoursSpecification = museum.openingHoursSpecification.map(ohs => {
+    return {
+      dayOfWeek: ohs.dayOfWeek,
+      closes: ohs.closes,
+      opens: ohs.opens
+    };
+  });
+  const newMuseum = Object.assign({}, museum, {openingHoursSpecification, logo: imageLd(museum.logo)});
+  delete newMuseum.twitterHandle;
+  return objToJsonLd(newMuseum, 'Museum');
+}
+
+function orgLd(org) {
+  return objToJsonLd({
+    name: org.name,
+    url: org.url,
+    logo: imageLd(org.logo),
+    sameAs: org.sameAs
+  }, 'Organization');
+}
+
+function personLd(person) {
+  return objToJsonLd({
+    name: person.name,
+    description: person.description,
+    image: person.image
+  }, 'Person');
+}
+
+function imageLd(image) {
+  return objToJsonLd({
+    url: image.contentUrl || image.url,
+    width: image.width,
+    height: image.height
+  }, 'ImageObject');
+}
```

Comment:
I am going to want to start using the thrift we've created, and then write serializers for this, but until we move to using Prismic, it's seems a little useless.